pytorch_blade/tests/test_config.py

```diff
@@ -162,6 +163,24 @@ def check_invalid(shapes):
             self.assertEqual(now_cfg.dynamic_tuning_shapes[1]['max'], shapes2['max'])
             self.assertEqual(now_cfg.dynamic_tuning_shapes[1]['opts'], shapes2['opts'])
 
+    def test_dynamic_tuning_inputs(self):
+        min_inp = {1: torch.tensor(1)}
+        max_inp = {3: torch.tensor(3)}
```

Comment:
Then I think the `1`, and `3`, are not shapes here.

Nit： use `torch.randn([1], device=self.device)` and `torch.randn([3], device=self.device)`

Reply:
device has been added:)